cshub-shared/src/api-calls/pages/GetPost.ts

```diff
@@ -15,6 +15,8 @@ export class GetPost implements IApiRequest {
     public static getURL: string = Requests.POSTDATA;
     public URL: string = GetPost.getURL;
 
-    constructor(public postHash: number) {}
+    constructor(public postHash: number) {
+        this.URL = this.URL.replace(/:hash/, postHash.toString());
```

Comment:
Not sure if this is the correct way to do things as now this method depends on the URL

Reply:
Hm perhaps that's the easiest way yes